src/main/resources/db/changelog/changes/04-create-user-roles-table.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-user-role-table
+      author: osa
+      changes:
+        - createTable:
+            tableName: user_roles
+            columns:
+              - column:
+                  name: user_id

```

Comment:
Add constraints to this table. Primary key (user_id, role_id) and foreign keys for both columns